README.md

```diff
@@ -73,43 +66,9 @@ $ npm test
 
 #### GCP (google cloud platform), automated app engine deployment
 
-1. Some examples (please visit the swagger documentation for more details)
-
-   1. Taxonomies -
-      1. all - https://mofb-api.appspot.com/api/v2/taxonomy,
-      2. by id - https://mofb-api.appspot.com/api/v2/taxonomy/{id}, e.g. 10
-      3. _Immediate_ subcategory of a taxonomy id (immediate children) - https://mofb-api.appspot.com/api/v2/taxonomy/{id}/children, e.g. 10
-      4. "Food" taxonomy and _all_ its sub categories - https://mofb-api.appspot.com/api/v2/taxonomy/food
-   2. Agencies -
-      1. all - https://mofb-api.appspot.com/api/v2/agency
-      2. by agency id - https://mofb-api.appspot.com/api/v2/agency/{id}, e.g. 5120
-      3. by taxonomy id - https://mofb-api.appspot.com/api/v2/agency?taxonomyId=11
-   3. Location -
-      1. by multiple agencies and taxonomy id https://mofb-api.appspot.com/api/v2/location?agencyId=39650,39651&taxonomyId=12
-      2. by location and service id, https://mofb-api.appspot.com/api/v2/location/39651/service/12
-
 ## Data
 
-1. Taxonomy: This dataset explains the taxonomy codes applicable to different types of
```

Comment:
Do we really not need these anymore? I haven't had a chance to examine the new datasets

Reply:
Yup. Technically, we were (/ are currently) using only 2 datasets - they had combined 1-6 into 7&8. We now have 2 different datasets, which have the same information but are laid out differently.